### ISSUE TITLE
[iOS] Unexpectedly found nil while unwrapping an Optional value

### ISSUE THREAD

**Issue by schmidtseb** [Rexios80/polar#11]
The complete error message reads:
`SwiftPolarPlugin.swift:173: Fatal error: Unexpectedly found nil while unwrapping an Optional value`

The issue is that `arguments` can contain no elements. My solution was to wrap everything in a `nil`-check and it is running again:
```dart
        if (arguments != nil) {
            let arguments = arguments as! [Any?]
            let feature = DeviceStreamingFeature(rawValue: arguments[0] as! Int)!
            let identifier = arguments[1] as! String
            
            self.streamingSubscriptions[feature]?[identifier]?.dispose()
        }
```

However, I then get an error within flutter in `model/polar_streaming.dart:39`:
`Unhandled Exception: type 'int' is not a subtype of type 'Map<String, dynamic>'`

The error occurs within:
```dart
  /// Create a [PolarAccData] from json
  PolarAccData.fromJson(this.identifier, Map<String, dynamic> json)
      : timeStamp = json['timeStamp'],
        samples =
            (json['samples'] as List).map((e) => Xyz.fromJson(e)).toList();
```

Here, `json['samples']` contains a list of integers instead of lists of lists of integers.  

I don't know yet if these are two issues or if they are related, though

**Comment by Rexios80** [comment]
Can I see what your code looks like? I'm pretty sure arguments should never be nil there.

**Comment by Rexios80** [comment]
I just released version 3.5.0 which should fix this
